include/FIFOSampleBuffer.h

```diff
@@ -91,85 +91,85 @@ class FIFOSampleBuffer : public FIFOSamplePipe
                      );
 
     /// destructor
-    ~FIFOSampleBuffer();
+    ~FIFOSampleBuffer() override;

```

Comment:
This seems to be a general issue in the project.
Did you consider submitting these changes upstream?
(Maybe also the cosmetics)

Reply:
It won't compile otherwise due to -Werror=inconsistent-missing-override

Reply:
yeah but that means it produces warnings upstream as well.
Maybe they'll just merge it when you add them.

Reply:
How about we merge this as is for now? Upstream repo doesn't seem to be particularly active, with prs being open for 2 month.

Reply:
Ok, it worked. Looks ugly tho...